monosi/anomalies/__init__.py

```diff
@@ -13,11 +13,11 @@ class AnomalyDetector:
 
 class ZScoreAnomalyDetector(AnomalyDetector):
     @classmethod
-    def _mean(cls, values: List[float]):
+    def _mean(cls, values: List[Any]):
```

Comment:
curious what the bug encountered was to change these back?

Reply:
I'm thinking about this now and maybe the method signature here should change and we should just force the values passed to definitely be floats. Problem is that .value member is technically Optional so Any or Optional[float] is correct type here when passing that in, but we should just get the list of floats and work off that - will update.

Reply:
So I kept the method signature float - basically what I think based on working with this code is that float needs to not be optional moving forward but outside scope here.